frontend/src/metabase/internal/components/Layout.jsx

```diff
@@ -106,8 +106,13 @@ export const InternalLayout = fitViewport(({ children }) => {
             </Link>
           </li>
           <li>
-            <Link className="link" to={"/_internal/colors"}>
-              <Label>Colors</Label>
+            <Link className="link" to={"/_internal/modals"}>
+              <Label>Modals</Label>
+            </Link>
+          </li>
+          <li>
+            <Link className="link" to={"/_internal/static-viz"}>
+              <Label>Static Visualizations</Label>
```

Comment:
Add a link to our `/_internal` route so we could access these pages more conveniently.
![image](https://user-images.githubusercontent.com/1937582/184914821-670b1bd5-0ff2-431b-b74f-9c7b4901e323.png)